doc/connectivity/bluetooth/bluetooth-le-host.rst

```diff
@@ -153,6 +153,117 @@ an application comes in direct contact with it only when using it in the
 so-called Connection-oriented Channels (CoC) mode. More information on
 this can be found in the :ref:`L2CAP API section <bt_l2cap>`.
 
+Terminology
+-----------
+
+Those definition are from the Core Specification version 5.4, volume 3, part A
+1.4.
+
+.. list-table::
+  :header-rows: 1
+
+  * - Term
+    - Description
+
+  * - Upper layer
+    - Layer above L2CAP, it exchange data in form of SDUs. It may be an
+      application or a higher level protocol.
+
+  * - Lower layer
+    - Layer below L2CAP, it exchange data in form of PDUs (or fragments). It is
+      usually the HCI.
+
+  * - Service Data Unit (SDU)
+    - Packet of data that L2CAP exchanges with the upper layer.
+
+      This term is relevant only Enhanced Retransmission mode, Streaming mode,
+      Retransmission mode and Flow Control Mode, not in Basic L2CAP mode.
+
+  * - Protocol Data Unit (PDU)
+    - Packet of data containing L2CAP data. PDUs always start with Basic L2CAP
+      header.
+
+      Types of PDUs for LE: :ref:`B-frames <bluetooth_l2cap_b_frame>` and
+      :ref:`K-frames <bluetooth_l2cap_k_frame>`.
+
+      Types of PDUs for BR/EDR: I-frames, S-frames, C-frames and G-frames.
+
+  * - Maximum Transmission Unit (MTU)
+    - Maximum size of an SDU that the upper layer is capable of accepting.
+
+  * - Maximum Payload Size (MPS)
+    - Maximum payload size that the L2CAP layer is capable of accepting.
+
+      Without segmentation or in Basic L2CAP mode, the MTU size is the same as
+      the MPS.
+
+  * - Basic L2CAP header
+    - Present at the beginning of each PDU. It contains two fields, the PDU
+      length and the Channel Identifier (CID).
+
+PDU Types
+---------
+
+.. _bluetooth_l2cap_b_frame:
+
+B-frame: Basic information frame
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+PDU used in Basic L2CAP mode. It contains a complete SDU as its payload.
```

Comment:
I think you either need ot remove that part in the table, or remove the second sentence
![image](https://github.com/zephyrproject-rtos/zephyr/assets/68428270/3ef5e7a0-9033-4daa-ab21-e323f0fd7b04)